.github/workflows/flake.yml

```diff
@@ -37,3 +42,23 @@ jobs:
           ignore: 'F821,W503,F841'
           max-line-length: '100'
           args: '--filename */SConscript, SConstruct'
+
+  # Run isort on the tree.
+  # This checks .py files only so misses SConstruct and SConscript files are not checked, rather
+  # for these files check them afterwards.  The output-filter will not be installed for this part
+  # so regressions will be detected but not annotated.
+  isort:
+    name: Isort check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.11
+      - uses: isort/isort-action@master
```

Comment:
Is this checked in?

Reply:
this is a public action, it doesn't need to be checked in.